packages/components/src/custom-select-control-v2/legacy-component/index.tsx

```diff
@@ -21,6 +22,7 @@ function CustomSelectControl( props: LegacyCustomSelectProps ) {
 		onChange,
 		size = 'default',
 		value,
+		className: classNameProp,
```

Comment:
Because I would otherwise get a `'className' is already declared in the upper scope` later in the file where `options` is mapped into `<CustomSelectItem />`s